Decred Wallet.xcodeproj/project.pbxproj

```diff
@@ -290,18 +286,27 @@
 			isa = PBXFrameworksBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
-				2506406E2304A2DE0028A9AC /* Dcrlibwallet.framework in Frameworks */,
 				B164982429AEDD7AF8C8782C /* Pods_Decred_Wallet.framework in Frameworks */,
+				25A0A7C1233E4FCC001064D5 /* Dcrlibwallet.framework in Frameworks */,
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 		};
 /* End PBXFrameworksBuildPhase section */
 
 /* Begin PBXGroup section */
+		25A0A7BB233E4D75001064D5 /* libs */ = {
```

Comment:
err, don't agree with this change. should leave the dcrlibwallet framework file untouched and the libs folder need not be linked to the project.

Reply:
resolved in 37e2453